### DISCUSSION TITLE
could we support  Gaussian/Laplace Differential Privacy on homoxgb histgram exchange?

### DISCUSSION THREAD

**Discussion by EasonFeng5870** [FederatedAI/FATE#4866] [Q&A]
hi there.
homo-sbt is using dh for random number generator seed. but there is paper <<Deep Leakage from Gradients>>  show that gradients reveal some properties of the training data, also show the Defense Strategies, like Noisy Gradients or Gradient Compression and Sparsification   or cryptography.

Is there a way we could support them?
